Automattic/wp-calypso
# Remove the state.sites.updates reducer

After #22617 and #22712, there is no longer anything interesting stored in `state.sites.updates`. The site updates data are equally well available as the `updates` property on a `state.sites.items` object. This PR does the following:

- move the selector `getUpdatesBySiteId` to `state/sites/selectors` and reads from `state.items`
- move the action creator `sitePluginUpdated` to `state/sites/actions`
- move the part of `updates` reducer that handles the `SITE_PLUGIN_UPDATED` action to the `sites.items` reducer
- move the separate schema definition for `updates` object to the `site` schema: it's just one of many site properties there
- move the tests to `state/sites` and drop the irrelevant ones

**How to test:**
1. Connect to a Jetpack site that has some updates (WordPress core or plugins)
2. Verify that it shows correct `SiteIndicator`
3. If there is a plugin update available, perform the update -- the updatable plugin count should drop to zero and the `SiteIndicator` should disappear.
